src/Diagnostics/define_diagnostics.jl

```diff
@@ -0,0 +1,156 @@
+
+    # Albedo
+    add_diagnostic_variable!(
+        short_name = "alpha",
+        long_name = "Albedo",
+        standard_name = "albedo",
+        units = "",
+        comments = "The fraction of incoming radiation reflected by the land surface.",
+        compute! = (out, Y, p, t) -> compute_albedo!(out, Y, p, t, land_model),
+    )
+
+    # Net radiation
+    add_diagnostic_variable!(
+        short_name = "rn",
+        long_name = "Net Radiation",
+        standard_name = "net_radiation",
+        units = "W m^-2",
+        comments = "Balance between incoming and outgoing shortwave and longwave radiation at the land surface.",
+        compute! = (out, Y, p, t) ->
+            compute_net_radiation!(out, Y, p, t, land_model),
+    )
+
+    # Surface temperature 
+    add_diagnostic_variable!(
+        short_name = "tsfc",
+        long_name = "Surface Temperature",
+        standard_name = "surface_temperature",
+        units = "K",
+        comments = "Temperature at the land-atmosphere interface.",
+        compute! = (out, Y, p, t) ->
+            compute_surface_temperature!(out, Y, p, t, land_model),
+    )
+
+    # Surface specific humidity
+    add_diagnostic_variable!(
+        short_name = "qsfc",
+        long_name = "Surface Specific Humidity",
+        standard_name = "surface_specific_humidity",
+        units = "",
+        comments = "Ratio of water vapor mass to total moist air parcel mass.",
+        compute! = (out, Y, p, t) ->
+            compute_surface_specific_humidity!(out, Y, p, t, land_model),
+    )
+
+    # Latent heat flux
+    add_diagnostic_variable!(
+        short_name = "lhf",
+        long_name = "Latent Heat Flux",
+        standard_name = "latent_heat_flux",
+        units = "W m^-2",
+        comments = "Exchange of energy at the land-atmosphere interface due to water evaporation.",
+        compute! = (out, Y, p, t) ->
+            compute_latent_heat_flux!(out, Y, p, t, land_model),
+    )
+
+    # Aerodynamic resistance 
+    add_diagnostic_variable!(
+        short_name = "rae",
+        long_name = "Aerodynamic Resistance",
+        standard_name = "aerodynamic_resistance",
+        units = "m s^-1",
+        comments = "Effiency of turbulent transport controlling the land-atmosphere exchange of sensible and latent heat.",
+        compute! = (out, Y, p, t) ->
+            compute_aerodynamic_resistance!(out, Y, p, t, land_model),
+    )
+
+    # Sensible heat flux
+    add_diagnostic_variable!(
+        short_name = "shf",
+        long_name = "Sensible Heat Flux",
+        standard_name = "sensible_heat_flux",
+        units = "W m^-2",
+        comments = "Exchange of energy at the land-atmosphere interface due to temperature difference.",
+        compute! = (out, Y, p, t) ->
+            compute_sensible_heat_flux!(out, Y, p, t, land_model),
+    )
+
+    # Vapor flux
+    add_diagnostic_variable!(
+        short_name = "vflux",
+        long_name = "Liquid water evaporation",
```

Comment:
"Water vapor flux"? since this could be sublimation too

Reply:
also as a comment - this is converted from a mass flux into an equivalent volume flux of liquid water. maybe this is what you meant

Reply:
`"Flux of water from the land surface to the atmosphere. E.g., evaporation or sublimation."`
Not sure if I should be e.g., or i.e., I guess there's no transpiration in bucket, so i.e., may be ok (if these are the only 2 options)

the units are `"m s^-1"`, correct? I guess this implies this is a volume of water so no need to specify in comments?